bin/eslint_d.js

```diff
@@ -31,7 +31,16 @@ if (cmd === 'start') {
   } else {
     var commands = ['stop', 'status', 'restart'];
     if (commands.indexOf(cmd) === -1) {
-      client.lint(process.argv.slice(2));
+      var concat = require('concat-stream');
```

Comment:
So the reason why I'm putting require statements into conditionals in this project (and in this project only), is because that is what makes eslint slow and I want to add as little overhead as possible. In this case, you could move it into the `if (stdin)` block or, even better, don't use a dependency at all ;-)